### PR TITLE
DOC-981: 5.10.1 Release Notes

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-981 

Description of Changes:
* Adding Release notes for TinyMCE 5.10.1 Enterprise

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes5101.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+**Enhanced Media Embed** 2.2.6 fixes the `mediaembed` content CSS loading incorrectly, which caused it to not display in previews.
+
+For information on the Enhanced Media Embed plugin, see: [Enhanced Media Embed plugin]({{site.baseurl}}/plugins/premium/mediaembed/).
+
+### Page Embed 1.1.1
+
+The {{site.productname}} 5.10.1 release includes an accompanying release of the **Page Embed** premium plugin.
+
+**Page Embed** 1.1.1 fixes a bug where the wrong media type used in the source URL dialog field.
```
Comment (lnewson) [inline release-notes/release-notes5101.md:46]:
```suggestion
**Page Embed** 1.1.1 fixes a bug where the wrong media type was used in the source URL dialog field.
```

File: release-notes/release-notes5101.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+- Tables could have an incorrect height set on rows when rendered outside of the editor.
+- In certain circumstances, the table of contents plugin would incorrectly add an extra empty list item.
+- The insert table grid menu displayed an incorrect size when re-opening the grid.
+- The word count plugin was treating the zero width space character (`&#8203;`) as a word.
+
+## Security fixes
+
+{{site.productname}} 5.10.1 provides fixes for the following security issues:
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ serverity vulnerability fixes.
```
Comment (lnewson) [inline release-notes/release-notes5101.md:131]:
```suggestion
All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
```

File: release-notes/release-notes5101.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+- The Spell Checker Pro plugin, see: [Spell Checker Pro plugin]({{site.baseurl}}/plugins/premium/tinymcespellchecker/).
+- The Link Checker plugin, see: [Link Checker plugin]({{site.baseurl}}/plugins/premium/linkchecker/).
+- The Image Tools plugin, see: [Image Tools plugin]({{site.baseurl}}/plugins/opensource/imagetools/).
+- The Enhanced Media Embed plugin, see: [Enhanced Media Embed plugin]({{site.baseurl}}/plugins/premium/mediaembed/).
+- Deploying the server-side components, see: [Server-side component installation]({{site.baseurl}}/enterprise/server/).
+
+### Security related dependency update for all 3 server-side services
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ serverity vulnerability fixes.
```
Comment (lnewson) [inline release-notes/release-notes5101.md:86]:
```suggestion
All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
```

**Review by lnewson — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

File: _new_content_templates/blank_releasenotes.md [hunk 4]
```diff
@@ -128,7 +128,11 @@ The {{site.productname}} vnumtxt release includes accompanying changes affecting
 - The Link Checker plugin (`linkchecker`)
 - The Spell Checker Pro plugin (`tinymcespellchecker`)
 
-The Java server-side components (`ephox-spelling.war`, `ephox-hyperlinking.war`, and `ephox-image-proxy.war`) have been updated to **version X.Y.Z**.
+The Java server-side components have been updated to the following versions:
+
+- `ephox-spelling.war`: X.Y.Z
```
Comment (EkimChau) [inline _new_content_templates/blank_releasenotes.md:133]: 2.118.1

File: release-notes/release-notes5101.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
+
+For information on the server-side components updates, see: [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges).
+
+## Known issues
+
+This section describes issues that users of {{site.productname}} 5.10.1 may encounter and possible workarounds for these issues.
+
+- [](#)
```
Comment (EkimChau) [inline release-notes/release-notes5101.md:139]: We need these placeholders removed on the final merge

**Review by EkimChau — APPROVED** [review]

File: release-notes/release-notes5101.md (hunk 5 above)
Comment (lnewson) [reply release-notes/release-notes5101.md]: The known issues section has been removed now.

**Pull request merged**